be/src/agent/task_worker_pool.cpp

```diff
@@ -960,8 +960,10 @@ void* TaskWorkerPool::_clear_transaction_task_worker_thread_callback(void* arg_t
         vector<string> error_msgs;
         TStatus task_status;
 
-        worker_pool_this->_env->storage_engine()->clear_transaction_task(
-            clear_transaction_task_req.transaction_id, clear_transaction_task_req.partition_id);
+        if (clear_transaction_task_req.transaction_id > 0) {
```

Comment:
Add comment to explain why there will be transaction id less than 0.

Reply:
OK